TestAssets/DesktopTestProjects/AppWithProjTool2Fx/App.csproj

```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Sdk">
-      <Version>1.0.0-alpha-20161102-2</Version>
+      <Version>1.0.0-alpha-20161104-02</Version>
```

Comment:
@srivatsn the original mail with SDK version number said `02`, so went with it. at first.